src/ensembl/production/dbcopy/api/urls.py

```diff
@@ -49,6 +49,7 @@
     re_path(r'transfers/(?P<job_id>[^/.]+)$', viewsets.TransferLogView.as_view(), name='transfers-list'),
     re_path(r'databases/(?P<host>[\w-]+)/(?P<port>\d+)', ListDatabases.as_view(), name='databaselist'),
     re_path(r'tables/(?P<host>[\w-]+)/(?P<port>\d+)/(?P<database>\w+)', ListTables.as_view(), name='tablelist'),
+    re_path(r'dcserversnames', ListDataCheckAllowedHost.as_view(), name='dcserversnames'),
```

Comment:
Could you change this to "dcservers" 
```suggestion
    re_path(r'dcservers', ListDataCheckAllowedHost.as_view(), name='dcservers'),
```